scripts/release.py

```diff
@@ -123,13 +123,13 @@ def update_dagster_cloud_pex(
         "PyGithub",
         "-o=dagster-cloud.pex",
         *platform_args,
-        "--platform=macosx_12_0_x86_64-cp-311-cp311",
-        "--platform=macosx_12_0_arm64-cp-311-cp311",
```

Comment:
i had to take these out to keep us below 100MB :/ is there a reason to keep them other than passing the local tests? I wouldn't expect anybody to actually be running this pex in a mac OS context since it's for github actions

Reply:
(well, they do have macOS github runners, but it would be very surprising if people were using them for this purpose)

Reply:
updated the PR to run the tests in docker images